common/device-model.cpp

```diff
@@ -2120,7 +2120,15 @@ namespace rs2
                                                   << "No match found for requested resolution: " << requested_res.first
                                                   << "x" << requested_res.second );
                     }
-                    sub->ui.selected_res_id = int(res_id);
+                    if (!sub->ui.selected_res_map_used)
+                        sub->ui.selected_res_id = int(res_id);
```

Comment:
consider `static_cast` (I know it's not your code :) )

Reply:
ok